main.go

```diff
@@ -90,7 +92,8 @@ type prometheus struct {
 func NewPrometheus() *prometheus {
 	conf, err := config.LoadFromFile(*configFile)
 	if err != nil {
-		glog.Fatalf("Error loading configuration from %s: %v", *configFile, err)
+		fmt.Fprintf(os.Stderr, "Error loading configuration from %s: %v\n", *configFile, err)
```

Comment:
It would be good to _also_ log this (error level). Some users (especially those used to the old way) might only look at logs, not at stdout.